admin/client/App/screens/List/index.js

```diff
@@ -333,25 +333,25 @@ const ListView = React.createClass({
 			checkedItems: {},
 		});
 	},
-	deleteTableItem (item, e) {
-		if (e.altKey) {
+	deleteTableItem (item, event) {
+		if (event.altKey) {
 			this.props.dispatch(deleteItem(item.id));
 			return;
 		}
 
-		e.preventDefault();
+		event.preventDefault();
 
 		this.setState({
 			confirmationDialog: {
 				isOpen: true,
 				label: 'Delete',
 				body: (
-					<p>
+					`<p>
 						Are you sure you want to delete <strong>${item.name}</strong>?
```

Comment:
The real solution to #3463 would be to remove this `$`, instead of all of that backtick business. Would you mind reverting those changes and removing the `$` here? Thanks!